demisto_sdk/commands/content_graph/objects/integration.py

```diff
@@ -96,6 +96,14 @@ def to_raw_dict(self) -> Dict:
         remove_nulls_from_dictionary(command)
         return command
 
+    def get_outputs_context_path(self) -> list:
+        """
+        This method returns a list of the all command's outputs context path.
+        """
+        outputs_context_path_list: List[Optional[str]] = []
+        outputs_context_path_list.extend(output.contextPath for output in self.outputs)
+        return outputs_context_path_list
```

Comment:
return `list(more_itertools.chain.from_iterable((output.contextPath for output in self.outputs)))`

Reply:
Done